### PR TITLE
SAK-33391 If an unsupported file type is attempted, ignore it.

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties [hunk 1]
```diff
@@ -43,4 +43,4 @@ link_to = Link to:
 load_thumbnail = Load thumbnail:
 link_button_tooltip = Insert a link
 image_button_tooltip = Insert an image
-choose_file = Choose file:
+choose_image_file = Choose an image file. It must be a png, gif or jpeg:
```
Comment (ottenhoff) [inline commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: please use oxford comma

File: library/src/morpheus-master/sass/modules/tool/commons/_commons.scss [hunk 2]
```diff
@@ -245,3 +245,9 @@
         padding-right: 3px;
     }
 }
+
+#commons-imagefile-instruction {
+    font-weight: bold;
+    font-size: bigger;
```
Comment (ottenhoff) [inline library/src/morpheus-master/sass/modules/tool/commons/_commons.scss]: is "bigger" valid ?  

https://developer.mozilla.org/en-US/docs/Web/CSS/font-size

File: commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties (hunk 1 above)
Comment (adrianfish) [reply commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: Are you taking the piss? The oxford comma does not apply here. The meaning is obvious.

File: library/src/morpheus-master/sass/modules/tool/commons/_commons.scss (hunk 2 above)
Comment (adrianfish) [reply library/src/morpheus-master/sass/modules/tool/commons/_commons.scss]: No. It should be larger. Bigly larger.

File: commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties (hunk 1 above)
Comment (ottenhoff) [reply commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: It's software consistency ..... we use oxford comma everywhere else. why would your new text be different?

Comment (adrianfish) [reply commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: I thought you were having a laugh. It's been a long day.

Comment (jonespm) [reply commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: I think being funny would have requested use of the Cambridge or Lancaster comma

Comment (adrianfish) [reply commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties]: We need a Sorbonne comma, too, or just a general 'Old Europe comma' for all the euro translators to use :)

**Pull request merged**